### ISSUE TITLE
Is this repository active?

### ISSUE THREAD

**Issue by atodorov** [bugzilla/docker-bugzilla-dev#3]
I see a docker image which is 3 years old and upon trying it out I get a bunch of random failures. 

First I was getting a 500 b/c the DB was unreachable.

Then I tried `bugzilla_config.sh` inside the running container which seems to create a DB schema (which I thought was already created) but still failures.

**Comment by dylanwh** [comment]
yeah this one is defunct, and it was never very good. There's an actively maintained docker config on https://github.com/mozilla-bteam/bmo and I'm working to get that running in https://github.com/bugzilla/harmony.
